src/reducers/project-state.js

```diff
@@ -435,10 +435,21 @@ const onLoadedProject = (loadingState, canSave, success) => {
         default:
             return;
         }
+    } else {
```

Comment:
Can this be extracted into a different PR?

Reply:
Did so -- see https://github.com/LLK/scratch-gui/pull/5291